brave-bom/pom.xml

```diff
@@ -184,6 +184,11 @@
         <artifactId>brave-instrumentation-jms</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>brave-instrumentation-jakarta-jms</artifactId>
```

Comment:
Just noticed the inconsistency with naming here (100% my bad) :( we could probably rename + relocate, @jcchavezs @shakuzen what do you think folks?

Reply:
Probably better to relocate to `jakarta-jms`

Reply:
@jcchavezs done 

```
[WARNING] The artifact io.zipkin.brave:brave-instrumentation-jakarta-jms:jar:5.15.1-SNAPSHOT has been relocated to io.zipkin.brave:brave-instrumentation-jms-jakarta:jar:5.15.1-SNAPSHOT
```